test/e2e/e2e.go

```diff
@@ -175,14 +201,74 @@ var _ = Describe("validation", func() {
 			})
 		})
 	})
+
+	Context("Creating AddressPool", func() {
+		It("Should create addresspool CR successfully", func() {
```

Comment:
Can you convert this to a table as I commented in https://github.com/metallb/metallb-operator/pull/27#discussion_r656123037 and add a case where autoassign is true (I believe that's the only case where the cm will be rendered differently).

Reply:
Hmm didn't u also said no need to over engineer this since we don't have additional CRDs ? I feel I am spending way too much time going back and forth on this ....

Reply:
No, I am asking a different thing. I am asking testing different configuration of the same CRD which is addresspool.